.github/workflows/merge_to_master.yaml

```diff
@@ -0,0 +1,82 @@
+name: update_upgrade_image
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
@jyejare do we want this github action to run only when we push changes to master branch and not others?
This means we will only be creating/updating `upgrade:latest` image and not any other `upgrade:x.y.z` ones.

Reply:
Yes ! Cause we are building the containers for robottelo/satellite-upgrade only from master branch.

Reply:
We just updated this in robottelo too, thanks for raising it @jameerpathan111. Easy to update later if we do want to start building z-stream branches for this repo.